mentoring-wg/gsoc-org-admin-guide.md

```diff
@@ -0,0 +1,163 @@
+# CNCF GSoC admin guide
+
+## Introduction
+
+> **Note**
+> This is a guide for Google Summer of Code admins at CNCF.
+
+While Google [defines](https://developers.google.com/open-source/gsoc/help/responsibilities#org_admin_responsibilities) responsibilities for organization admins and gives some [tips](https://developers.google.com/open-source/gsoc/help/oa-tips), there are variations in how organizations run the program suitable for their communities.
+
+## Outline of responsibilities
+
+First of all, GSoC admins are required to read these documents from Google:
+
+* [GSoC admin responsibilities](https://developers.google.com/open-source/gsoc/help/responsibilities#org_admin_responsibilities)
+* [GSoC admin tips](https://developers.google.com/open-source/gsoc/help/oa-tips)
+
+In summary, responsibilities can be outlined as following.
+
+* Recruitment: GSoC admins are responsible for recruiting mentors from the CNCF communities and also contributors to participate in the program. They reach out to communities, promote the program, and answer questions from interested parties.
```

Comment:
I feel like `CNCF communities` is a little vague here. Aren't mentors supposed to come directly from projects, and thus be either regular contributors or maintainers of CNCF projects? If so, I'd say that specifically, especially since the following sentence then states that mentors "reach out to communities", etc.

Reply:
Just thinking... Maybe it is a better idea to use "mentee" instead of "contributor" in the entire document...

In GSoC terms, "contributor" is the person who is getting the mentorship from the mentor. I was using that term throughout this document, but using "mentee" would be less confusing I think.

What do you think @natalisucks @nate-double-u

Reply:
@aliok I completely agree that using "mentee" would be less confusing for all

Reply:
I'm of two minds on this one, yes "mentee" would probably be less confusing, but I think most of the GSoC docs use the term "contributor." I like the idea of using the same terminology as GSoC docs in this doc as it's specifically about our Admin responsibilities as it relates to GSoC. So I think we should go with "contributor," with maybe a short explainer early in the page.

Reply:
I didn't change the usage of "contributor" but added a note at the top of the document:

> **Note** 
> The "contributor" term throughout this document is used as "mentee". Please do not mix the term "contributor" with regular contributors and maintainers of the CNCF projects. The term "contributor" is used to follow Google's terminology for the GSoC program.